test/bats/test.bats

```diff
@@ -83,7 +83,30 @@ SLEEP_TIME=1
     wait_for_process ${WAIT_TIME} ${SLEEP_TIME} kubectl delete pod sbom --namespace default
 }
 
-@test "sbom/notary/cosign/licensechecker verifiers test" {
+@test "schemavalidator verifier test" {
+    run kubectl apply -f ./library/default/template.yaml
+    assert_success
+    sleep 5
+    run kubectl apply -f ./library/default/samples/constraint.yaml
+    assert_success
+    sleep 5
+
+    run kubectl apply -f ./config/samples/config_v1alpha1_verifier_schemavalidator.yaml
+    sleep 5
+    run kubectl run schemavalidator --namespace default --image=wabbitnetworks.azurecr.io/test/all-in-one-image:signed
```

Comment:
it is best to use an image with individual artifact types ( instead of all in one) so we can isolate the failure. Could we add a todo note to replace this image once we have local private registry for our test setup?

Reply:
I will make this change.